### DISCUSSION TITLE
FID calculations in paper

### DISCUSSION THREAD

**Discussion by YotamNitzan** [PixArt-alpha/PixArt-alpha#60] [General]
The paper reports FID with COCO (e.g., Table 2, Figure 20). What PixArt model was used in these experiments? 
My best guess is that it's a model operating in 512x512, which is more compatible with previous models and the COCO dataset. Is that true or did you use the 1024x1024 model?

Thanks!

**Comment by lawrence-cj** [comment]
We use 256px model of course. Since everyone in the community seems to use the 256 model for COCO FID calculation.

**Reply by YotamNitzan** [reply]
Thanks! And would you be able to release that model to allow others to produce quantitative comparisons in future papers?

**Reply by lawrence-cj** [reply]
Will release once ready.

**Reply by Gus-Guo** [reply]
Hi lawrence, what kind of feature do you used to compute the FID scores? Do you use inception feature that is pretrained on ImageNet or CLIP feature?
